### PR TITLE
remove Alpha

### PR DESCRIPTION
**Pull request by KillariDev**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+# The Interceptor
+The Interceptor is a powerful tool for simulating transactions on EVM-based chains, including Ethereum Mainnet, Base, Optimism, Gnosis, and more. It offers clear, intuitive explanations for the transactions you initiate. The Interceptor can seamlessly complement your browser wallet, like MetaMask, or function as a standalone transaction simulation tool.
+
+With MetaMask integration, you can forward transactions for signing and send them effortlessly. What truly sets The Interceptor apart, however, is its Simulation Mode. This feature lets you simulate multiple transactions simultaneously, providing a detailed preview of their outcomes-even enabling you to interact with DApps at no cost!
```
Comment (MicahZoltu) [inline readme.md:6]:
```suggestion
With MetaMask integration, you can forward transactions for signing and send them effortlessly. What truly sets The Interceptor apart, however, is its Simulation Mode. This feature lets you simulate multiple transactions sequentially, providing a detailed preview of their outcomes-even enabling you to interact with DApps at no cost!
```

File: readme.md [hunk 2]
```diff
@@ -1,25 +1,37 @@
...
-
-<img src = "transaction_outcome.png" alt = "The cutest dino" style = "width: 400px;"/>
 <img src = "popup_view.png" alt = "The cutest dino" style = "width: 300px;"/>
 
-# Privacy
-Your privacy is our priority. The Interceptor is purpose-built to minimize data leakage: no external queries are made without direct user input. By default, it connects to Ethereum RPC nodes operated by [Dark Florists (us)](https://www.dark.florist/), though you can configure it to connect to other RPC nodes if desired. Please note, The Interceptor requires RPC nodes that support the mandatory `eth_simulateV1` endpoint, currently available in Geth and Nethermind.
+## Privacy
+Your privacy is our priority. The Interceptor is purpose-built to minimize data leakage: no external queries are made without direct user input. By default, it connects to Ethereum RPC nodes operated by [Dark Florist (us)](https://www.dark.florist/), though you can configure it to connect to other RPC nodes if desired. Please note, The Interceptor requires RPC nodes that support the `eth_simulateV1` endpoint, currently available in Geth and Nethermind.
```
Comment (MicahZoltu) [inline readme.md:24]:
```suggestion
Your privacy is our priority. The Interceptor is purpose-built to minimize data leakage: no external queries are made without direct user input except for calls to the configured Ethereum RPC server. By default, it connects to Ethereum RPC nodes operated by [Dark Florist (us)](https://www.dark.florist/) which doesn't log requests, though you can configure it to connect to other RPC nodes if desired. Please note, The Interceptor requires RPC nodes that support the `eth_simulateV1` endpoint, currently available in Geth and Nethermind.
```

File: readme.md [hunk 3]
```diff
@@ -1,25 +1,37 @@
...
+
+## Using The Interceptor on various EVM chains
+We are not currently operating nodes for Base, Optimism etc, in order to use these chains, add your own RPC.
```
Comment (MicahZoltu) [inline readme.md:27]:
```suggestion
We are not currently operating nodes for Base, Optimism etc, in order to use these chains you will need to add your own RPC.
```

File: readme.md [hunk 4]
```diff
@@ -1,25 +1,37 @@
...
+
+## Good tools to use with The Interceptor
+The Interceptor requires websites to be decentralized and only use users own wallet to access the network. The following tools are tested to work on The Interceptor:
```
Comment (MicahZoltu) [inline readme.md:30]:
```suggestion
The Interceptor requires websites to be decentralized and only use users' own wallet to access the network. The following tools are tested to work on The Interceptor:
```

File: readme.md [hunk 5]
```diff
@@ -1,25 +1,37 @@
...
+- [Lunaria](https://lunaria.dark.florist/)
+- [NFT sender](https://nftsender.dark.florist/)
+- [Horswap 2](https://2.horswap.eth.limo/)
+- [Bouquet](https://bouquet.dark.florist/)
```
Comment (MicahZoltu) [inline readme.md:34]: Should we add PetalLock to the list?

**Review by MicahZoltu — APPROVED** [review]

File: readme.md (hunk 5 above)
Comment (KillariDev) [reply readme.md:34]: added

**Pull request merged**